portals/podaac-cloud/config.json

```diff
@@ -1,19 +1,27 @@
 {
+  "description": "The Physical Oceanography Distributed Active Archive Center (PO.DAAC) is a NASA Earth Observing System Data and Information System (EOSDIS) data center managed by the Earth Science Data and Information System (ESDIS) Project.",
```

Comment:
Not sure if it's worth adding something about how this one has cloud_hosted holdings, since this is the same description as PODAAC.